### ISSUE TITLE
Improve documentation on Capsule Proxy

### ISSUE THREAD

**Issue by maxgio92** [clastix/capsule#640]
This issue tracks the need to add details on how to use Capsule Proxy, specifically a user guide.

The guide could add details about how to communicate over the proxy, instead of directly to the API server.

As a proposal, a specific how-to guide could be added.

**Comment by wasup-yash** [comment]
hey @maxgio92, I can work on this, can you provide the resources from where the current users taking references for the "HOW TO" part so I can document it in a `how-to guide` as you mentioned.

**Comment by maxgio92** [comment]
Great @wasup-yash, thank you!

**Comment by wasup-yash** [comment]
> Great @wasup-yash, thank you! You can refer to [`docs/content/general/proxy.md`](https://github.com/clastix/capsule/blob/master/docs/content/general/proxy.md) for the Capsule Proxy general documentation. Let me know if I can help you further :)

Thanks for the reference, I am currently understanding the Capsule-proxy landscape on how it fit into the picture for the new users and then we can work on the introductory guide for it, regarding this I have a question like we need to add the guide like apart from the current documented Capsule-proxy docs right? and also I have some suggestions like we can improve the current documentation of the Capsule proxy to get it more organized so we can work on it first and then we can create a guide, thoughts?

**Comment by maxgio92** [comment]
Hey @wasup-yash, I don't have strong opinions. I think the current guide could be taken as starting point to be improved.
For example answering questions like:
* use cases for which is needed
* further value it can provide, besides Capsule
Then, the installation I think can be expanded.

All of this could be part of a dedicated section I think, more than a single page - like now.

In any case I think it makes sense your suggestion, to starting improving the organization of the current single-page one.
What do you think?

**Comment by wasup-yash** [comment]
> 



> Hey @wasup-yash, I don't have strong opinions. I think the current guide could be taken as starting point to be improved. For example, answering questions like:
> 
> * use cases for which is needed
> * further value it can provide, besides Capsule, even when not required for the use case.
> 
> Then, the installation I think can be expanded.
> 
> Going further, all of this could be part of a dedicated section I think, more than a single page - like now.
> 
> In any case, I think it makes sense your suggestion, to start improving the organization of the current single-page one. What do you think?

yeah, it makes sense to improve the current docs part and then go on with the further use case guides and have dedicated sections if needed (if we need them) as I also discussed in the meeting today with @prometherion .
